packages/xod-client/src/utils/components/Autoscroll.jsx

```diff
@@ -0,0 +1,98 @@
+/*
+  Edited fork of `autoscroll-react` from
+  https://github.com/xodio/autoscroll-react.git#cbb3422461a28b40ca082d6a5f33f667508c3288
```

Comment:
Nope. The original should be the original original. And a note on what’s been changed:

- onScrollFromBottom
- scrollDown